src/main.py

```diff
@@ -0,0 +1,56 @@
+# OMIT !/usr/bin/env python3
+from pathlib import Path
+import click
+from io import StringIO
+
+from src.option_select import select
+from src.run_cocci import run_cocci
+
+@click.command
+@click.option(
+    "-i",
+    "--input",
+    type=Path,
+    required=True,
+    help="Directory containing your C source files.",
+)
+@click.option(
+    '--scope',
+    type=click.Choice(['all', 'select']), 
+    prompt='Choose an option to compare all or just a single file'
+)
+@click.option(
+    "-v",
+    "--verbosity", 
+    type=str,
+    default=0,
+    required=False,
+    help = "Enter 0 or 1"
+)
+def cli(input, scope, verbosity):
+    if not input.is_dir():
+        raise Exception(f"input path is not valid")
+
+    patch_list = (Path.cwd() / "cocci").glob("*.cocci")
```

Comment:
The user could call the command from outside of the project, so you can't use `cwd()` here. Take a look at how using `__file__`  can be of help here